keyboards/explorer_pi/3x6/rules.mk

```
@@ -0,0 +1,2 @@
+SPLIT_KEYBOARD = yes
+SERIAL_DRIVER = vendor
```

move `SPLIT_KEYBOARD` to `info.json`
```suggestion
SERIAL_DRIVER = vendor

```